config/routes.rb

```diff
@@ -1,17 +1,14 @@
 Rails.application.routes.draw do
-  get 'password_reset/new'
-  get 'password_reset/edit'
-  get 'sessions/new'
-  get 'users/new'
-  root 'static_pages#home'
-  get '/help', to: 'static_pages#help'
-  get '/about', to: 'static_pages#about'
-  get '/contact', to: 'static_pages#contact'
-  get '/signup', to: 'users#new'
+  root   'static_pages#home'
+  get    '/help',    to: 'static_pages#help'
+  get    '/about',   to: 'static_pages#about'
+  get    '/contact', to: 'static_pages#contact'
+  get    '/signup',  to: 'users#new'
   get    '/login',   to: 'sessions#new'
   post   '/login',   to: 'sessions#create'
   delete '/logout',  to: 'sessions#destroy'
   resources :users
-  resources :account_activations, only: [:edit]
-  resources :password_resets,     only: [:new, :create, :edit, :update]
+  resources :account_activations, only: %s(edit)
```

Comment:
```suggestion
  resources :account_activations, only: %i(edit)
```
FYI
他と揃えてこれでも問題ないですよ！